BGAnimations/ScreenSystemLayer overlay.lua

```diff
@@ -208,4 +208,38 @@ t[#t+1] = LoadFont("Common Footer")..{
 	end
 }
 
-return t
\ No newline at end of file
+-- Date & time at lower-center of screen

```

Comment:
I think we should at least mention that this won't overlap with the above actor because that one isn't displayed on either of the ScreenEval screens. Better yet would be to wrap these actors in an if/else to guarantee it but I think it's fine.

Reply:
I updated the comment to include that information. I think wrapping the two actors in if/else is not possible: The overlay is created _once_ for the whole game. Adding/removing actors is as far as I know not possible after initialization, so all the actors have to be there from the start. That's why we have to resort to hiding elements that are not relevant for any given screen.